data/json/monster_special_attacks/monster_attacks.json

```diff
@@ -323,9 +323,9 @@
     "damage_max_instance": [ { "damage_type": "bash", "amount": 20 } ],
     "hit_dmg_u": "%1$s stares at you intently and a sudden wind buffets you!",
     "hit_dmg_npc": "%1$s stares intently at <npcname> and a hurricane force wind hits them!",
-    "no_dmg_msg_u": "%1$s's wind buffets your %2$s, but it doesn't catch hold.",
-    "no_dmg_msg_npc": "%1$s's winds hit <npcname>, but it doesn't catch hold.",
-    "miss_msg_u": "%s stares at you, but you the wind bursts past you!",
+    "no_dmg_msg_u": "%1$s's wind buffets your %2$s, but it doesn't catch hold!",
```

Comment:
So I don't char about exclamation points or not but lets follow whatever the accepted standard is.

Reply:
Oops I messed up rebasing good catch